gsql/cache/cache_key.py

```diff
@@ -0,0 +1,74 @@
+from functools import lru_cache
+from key_pair import Keypair
+from gsql.cache.cache_handler import (
+    CacheDict,
+    check_json_file_exists,
+    count_lines_in_json,
+    create_json_file,
+    keys_exists,
+    read_from_json,
+    remove_key_from_json,
+    write_to_json,
+)
+from gsql.cache.constant import MAXIMUM_SIZE_OF_JSON_FILE
+
+
+"""
+    Cachekey contains the type of query,
+    the query itself, database id and the sheet id.
+"""
```

Comment:
Please write doc strings at the starting of the __init__ method.

Reply:
Yes, done